src/views/teachers/landing/landing.jsx

```diff
@@ -338,7 +338,21 @@ const Landing = props => (
                         <FormattedMessage id="teacherlanding.accountsTitle" />
                     </h2>
                     <p>
-                        <FormattedHTMLMessage id="teacherlanding.accountsDescription" />
+                        <FormattedMessage
```

Comment:
That was the only place where FormattedHTMLMessage was used - so you can remove it

Reply:
Thanks! That probably explains the lint error too!